statsmodels/stats/_lilliefors.py

```diff
@@ -216,21 +313,36 @@ def kstest_normal(x, pvalmethod='approx'):
     '''
 
     x = np.asarray(x)
-    z = (x-x.mean())/x.std(ddof=1)
-    nobs = len(z)
+    nobs = len(x)
+
+    if dist == 'norm':
+        z = (x-x.mean())/x.std(ddof=1)
+        test_d = stats.norm.cdf
+        lilliefors_table = lilliefors_table_norm
+    elif dist == 'exp':
+        z = x/x.mean()
+        test_d = stats.expon.cdf
+        lilliefors_table = lilliefors_table_expon
+        pvalmethod = 'table'
+    else:
+        print('test distribution must be norm or exp.')
```

Comment:
Thise should raise an exception, no print in code
ValueError or TypeError, I don't remember